### PR TITLE
Changed Folder Structure for editing and viewing a blog

### PR DESCRIPTION
**Pull request by 2005balaji**

### CODE REVIEW

File: apps/web/pages/blog/blog.[id].vue
Comment (Arpit1204) [inline apps/web/pages/blog/blog.[id].vue:1]: Why blog.[id].vue??

Comment (2005balaji) [reply apps/web/pages/blog/blog.[id].vue:1]: hey thx for pointing out ,

- changed file name to [id].vue

**Review by Arpit1204 — APPROVED** [review]
LGTM

**Comment by 2005balaji** [comment]
Editor Folder

**Pull request merged**